aspnetcore/razor-components/components.md

```diff
@@ -476,11 +477,11 @@ If `IsCompleted` is `false`, the check box is rendered as:
 
 **Additional information on Razor**
 
-For more information on Razor, see the [Razor syntax reference](https://docs.microsoft.com/aspnet/core/mvc/views/razor). Note that not all of the features of Razor are available in Razor Components at this time.
+For more information on Razor, see the [Razor syntax reference](xref:mvc/views/razor). Note that not all of the features of Razor are available with Razor Components at this time.
```

Comment:
~~Note that not all of the features of Razor are available with Razor Components at this time.~~